DIPs/DIP1337.md

```diff
@@ -0,0 +1,85 @@
+# Tail Call Optimization requirement
+
+| Section         | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | 1337                                                            |
+| Author:         | Dietrich Daroch (name dot surname on gmail)                     |
+| Implementation: | (links to implementation PR if any)                             |
+| Status:         | -                                                               |
+
+
+
+## Abstract
+
+Add an attribute that forces tail call optimization.
```

Comment:
Afaik everyone wants to reduce the already high load attribute variety in D, so therefore I would guess that giving out another attribute  like `@tco` is a something that is unlikely to happen, but I don't know so much about D's history.